src/docs/asciidoc/80-observability.adoc

```diff
@@ -122,7 +122,7 @@ To activate the instrumentation add the following runtime dependency to your pro
 [source, xml]
 ----
 <dependency>
-  <groupId>org.springframework.modulith</groupId>
```

Comment:
That's invalid, too, as the namespace for Spring Modulith starting with 1.0 will live in the `….modulith` namespace.

Reply:
But currently a user following the docs will not be able to access the artifact from `.modulith` group. Shouldn't the group be changed to `.modulith` across the board once version 1.0 is released?

Reply:
That's not Maven Central but our custom milestone repository, but yeah. Thanks!